server/asset/bip-id.go

```diff
@@ -3,594 +3,591 @@
-	"COS":         3077,     // Contentos
-	"DYN":         3381,     // Dynamic
-	"SEQ":         3383,     // Sequence
-	"DEO":         3552,     // Destocoin
-	"DST":         3564,     // DeStream
-	"NAS":         2718,     // Nebulas
-	"BND":         2941,     // Blocknode
-	"CCC":         3276,     // CodeChain
-	"ROI":         3377,     // ROIcoin
-	"IOTA":        4218,     // IOTA
-	"AXE":         4242,     // Axe
-	"FIC":         5248,     // FIC
-	"HNS":         5353,     // Handshake
-	"Stacks":      5757,     //
-	"SLU":         5920,     // SILUBIUM
-	"GO":          6060,     // GoChain GO
-	"BPA":         6666,     // Bitcoin Pizza
-	"SAFE":        6688,     // SAFE
-	"ROGER":       6969,     // TheHolyrogerCoin
-	"BTV":         7777,     // Bitvote
-	"BTQ":         8339,     // BitcoinQuark
-	"SBTC":        8888,     // Super Bitcoin
-	"NULS":        8964,     // NULS
-	"BTP":         8999,     // Bitcoin Pay
-	"NRG":         9797,     // Energi
-	"BTF":         9888,     // Bitcoin Faith
-	"GOD":         9999,     // Bitcoin God
-	"FO":          10000,    // FIBOS
-	"BTR":         10291,    // Bitcoin Rhodium
-	"ESS":         11111,    // Essentia One
-	"IPOS":        12345,    // IPOS
-	"BTY":         13107,    // BitYuan
-	"YCC":         13108,    // Yuan Chain Coin
-	"SDGO":        15845,    // SanDeGo
-	"ARDR":        16754,    // Ardor
-	"SAFE":        19165,    // Safecoin
-	"ZEL":         19167,    // ZelCash
-	"RITO":        19169,    // Ritocoin
-	"XND":         20036,    // ndau
-	"PWR":         22504,    // PWRcoin
-	"BELL":        25252,    // Bellcoin
-	"CHX":         25718,    // Own
-	"ESN":         31102,    // EtherSocial Network
-	"ThePower":    31337,    //
-	"TEO":         33416,    // Trust Eth reOrigin
-	"BTCS":        33878,    // Bitcoin Stake
-	"BTT":         34952,    // ByteTrade
-	"FXTC":        37992,    // FixedTradeCoin
-	"AMA":         39321,    // Amabig
-	"STASH":       49344,    // STASH
-	"KETH":        65536,    // Krypton World
-	"RYO":         88888,    // c0ban
-	"WICC":        99999,    // Waykichain
-	"AKA":         200625,   // Akroma
-	"GENOM":       200665,   // GENOM
-	"ATS":         246529,   // ARTIS sigma1
-	"X42":         424242,   // x42
-	"VITE":        666666,   // Vite
-	"ILT":         1171337,  // iOlite
-	"ETHO":        1313114,  // Ether-1
-	"XERO":        1313500,  // Xerom
-	"LAX":         1712144,  // LAPO
-	"BCO":         5249353,  // BitcoinOre
-	"BHD":         5249354,  // BitcoinHD
-	"PTN":         5264462,  // PalletOne
-	"WAN":         5718350,  // Wanchain
-	"WAVES":       5741564,  // Waves
-	"SEM":         7562605,  // Semux
-	"ION":         7567736,  // ION
-	"WGR":         7825266,  // WGR
-	"OBSR":        7825267,  // OBServer
-	"AQUA":        61717561, // Aquachain
-	"kUSD":        91927009, // kUSD
-	"FLUID":       99999998, // FluiChains
-	"QKC":         99999999, // QuarkChain
+var BipIDs = map[int32]string{
+	0:        "BTC",
+	1:        "Testnet",
```

Comment:
BTW, we need a solution for testnets

Reply:
Testnet is already handled on the backends using an argument to the constructor. The network types are defined in asset/common.go. I was thinking we could use those network IDs throughout.

Reply:
I guess this was mainly a note to self regarding recognizing testnet (or other) somehow in the DB.  It would be unpleasant if this was left to the database name with no other checks since the wrong DB could be use by accident.  There other places in DEX where similar checks will be necessary.